ibm_catalog.json

```diff
@@ -506,6 +523,23 @@
             {
               "key": "ibmcloud_api_key"
             },
+            {
+              "key": "provider_visibility",
```

Comment:
You have added `provider_visibility` twice

Reply:
@iamar7  I think you are referring to the provider_visibility added in configuration of both the DA s once for each

Reply:
@Aditya-ranjan-16 We don't need to expose the  `provider_visibility` twice in the `ibm_catalog.json` as in the UI it will show as repetition since both the variable has the same name so please remove this.